### PR TITLE
Remove plot_directive and use the one from Matplotlib

### LINKED ISSUE #26: Docrepr fails with Sphinx v4
It fails for multiple reasons:

- jsmath extension not available anymore: Attempt to fix this in https://github.com/spyder-ide/docrepr/pull/25
- Sphinx's `add_directive` does not accept the `alt` keyword argument

### LINKED ISSUE #29: Get rid of the Matplotlib's plot_directive copy
As mentioned in https://github.com/spyder-ide/docrepr/pull/28#issuecomment-988660335, there is a copy of [Matplotlib's `plot_directive`](https://github.com/matplotlib/matplotlib/blob/f6e0ee49c598f59c6e6cf4eefe473e4dc634a58a/lib/matplotlib/sphinxext/plot_directive.py) since [this commit](https://github.com/spyder-ide/docrepr/commit/7ce774ce592a80a7da23a4ac1b4f48e8d53f9e05). 

We should probably remove this copy and use Matplotlib's directive instead. This would reduce some maintenance burden, e.g. we wouldn't have to [update our copy for latest Sphinx version support](https://github.com/spyder-ide/docrepr/pull/28), as Matplotlib devs did it already.

### PR DESCRIPTION
**Pull request by martinRenou**
This fixes #29 and can replace https://github.com/spyder-ide/docrepr/pull/28

Also fix #26

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
Capture

😎

File: docrepr/sphinxext/__init__.py [hunk 1]
```diff
@@ -1 +1,11 @@
- 
+from warnings import warn
+
+warn(
+    """
+    docrepr.sphinxext.plot_directive is deprecated and will eventually be removed,
```
Comment (CAM-Gerlach) [inline docrepr/sphinxext/__init__.py]:
```suggestion
    docrepr.sphinxext.plot_directive is deprecated and will eventually be removed;
```

Fix comma splice

File: docrepr/sphinxext/__init__.py [hunk 2]
```diff
@@ -1 +1,11 @@
...
+    please use matplotlib.sphinxext.plot_directive instead
+    """,
+    DeprecationWarning
```
Comment (CAM-Gerlach) [inline docrepr/sphinxext/__init__.py]:
```suggestion
    DeprecationWarning,
```

Consistency

**Review by CAM-Gerlach — CHANGES_REQUESTED** [review]
Other than the two tiny nits, can you add a `plot` extra to the `setup.py`, requiring a recent enough version of Matplotlib with a Sphinx 4-compatible copy of the plot_directive? Thanks!

**Comment by CAM-Gerlach** [comment]
Makes sense to me; @ccordoba12 just to make sure, was there any reason you can recall that we had to vendor the plot directive?

**Comment by ccordoba12** [comment]
> @ccordoba12 just to make sure, was there any reason you can recall that we had to vendor the plot directive?

Probably because the directive was not part of Matplotlib at the time when I created this package.

File: docrepr/sphinxext/__init__.py [hunk 3]
```diff
@@ -1 +1,11 @@
- 
+from warnings import warn
+
+warn(
+    """
+    docrepr.sphinxext.plot_directive is deprecated and will eventually be removed;
+    please use matplotlib.sphinxext.plot_directive instead
+    """,
```
Comment (CAM-Gerlach) [inline docrepr/sphinxext/__init__.py]:
```suggestion
warn(
    "docrepr.sphinxext.plot_directive is deprecated and will eventually be "
    "removed; please use matplotlib.sphinxext.plot_directive instead",
```

As confirmed in my testing, the triple-quoted string literal here preserves line breaks and whitespace, thus messing with the formatting—sorry I didn't notice this before! This fixes it, which I've also tested.

**Review by CAM-Gerlach — DISMISSED** [review]
Looks good, thanks @martinRenou ! Just one issue with the warning that I didn't notice before, sorry.

**Comment by CAM-Gerlach** [comment]
Also, in my testing locally (shouldn't be hard to get things set up on the CIs as a smoke test, though we'll need Netlify and a bit of orchestration if we want actual rendered previews), running `test_plot()` in `fabfile.py` produces no output at all (not even in the HTML source) for the plot, despite the example being tested being identical to the one in the [Matplotlib docs for matplotlib.sphinxext.plot_directive](https://matplotlib.org/stable/api/sphinxext_plot_directive_api.html?highlight=plot%20directive#module-matplotlib.sphinxext.plot_directive).

image

Since there are no errors or warnings, I'm not sure why its not working. However, this at least ensures that all the other tests work fine with Sphinx 4 and isn't really a regression, since it doesn't work in Sphinx 3 either with or without this change, so I'm fine with merging it for now, but I wanted to bring that up in case you want to fix it here, or if there's another reason its not rendering; otherwise we should open an issue to track it. Thanks!

File: docrepr/sphinxext/__init__.py (hunk 3 above)
Comment (martinRenou) [reply docrepr/sphinxext/__init__.py]: Thanks!

**Comment by martinRenou** [comment]
That's odd. When running the [test_plot](https://github.com/spyder-ide/docrepr/blob/master/fabfile.py#L92) test I get the following result:

plotdirective

So it's not quite what you are getting. Are you sure you are looking at the output of the plot directive test?

The only issue I see on my side is that it's looking for the plot image under the `_images` directory, while it's located at the same level as the html file. But I can also reproduce this on `master`, so it doesn't seem to be a regression from this PR. I will open an issue for this specific problem. See #31

**Comment by CAM-Gerlach** [comment]
> So it's not quite what you are getting. Are you sure you are looking at the output of the plot directive test?

Yup, I tried it multiple times and varied the plotting code, as well as running it with `test_all()`, and saw no change in the behavior in my screenshot. I checked the source HTML and indeed, the `docstring` `<div>` is completely empty, and the directory contents were as follows:

image

The contents of `docstring.rst` indeed contained the plot directive passed as input:

```rst
.. plot::

   >>> import matplotlib.pyplot as plt
   >>> plt.plot([1,2,3], [4,5,6])
```

This rules out the browser, so remaining possibilities I can think of are:

* Something platform specific with temp directory handling, file copying, or otherwise (since it looks like you're on *nix of some sort, while I'm on Windows)
* Differences in our invocations `python -b -X dev -c "import fabfile; fabfile.test_plot()"` for me from the docrepr root dir; I get no warnings with the latest deps 
* Differences in our Python environments (I'm using Python 3.9.7 from Conda-Forge)
* Differences in our dependency stacks (to confirm, I created a fresh clean environment from `-c conda-forge` with just `docutils`, `jinja2`, `sphinx`, `matplotlib` and `ipython` as specs, plus `python=3.9`, and the issue did not change; here's my `conda list` output:

<details>

```
$ conda list
# packages in environment at C:\Miniconda3\envs\docrepr-env:
#
# Name                    Version                   Build  Channel
alabaster                 0.7.12                     py_0    conda-forge
babel                     2.9.1              pyh44b312d_0    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
brotli                    1.0.9                h8ffe710_6    conda-forge
brotli-bin                1.0.9                h8ffe710_6    conda-forge
brotlipy                  0.7.0           py39hb82d6ee_1003    conda-forge
ca-certificates           2021.10.8            h5b45459_0    conda-forge
certifi                   2021.10.8        py39hcbf5309_1    conda-forge
cffi                      1.15.0           py39h0878f49_0    conda-forge
charset-normalizer        2.0.9              pyhd8ed1ab_0    conda-forge
colorama                  0.4.4              pyh9f0ad1d_0    conda-forge
cryptography              36.0.0           py39h7bc7c5c_0    conda-forge
cycler                    0.11.0             pyhd8ed1ab_0    conda-forge
decorator                 5.1.0              pyhd8ed1ab_0    conda-forge
docrepr                   0.2.0.dev0                dev_0    <develop>
docutils                  0.15.2           py39hcbf5309_3    conda-forge
fonttools                 4.28.3           py39hb82d6ee_0    conda-forge
freetype                  2.10.4               h546665d_1    conda-forge
icu                       68.2                 h0e60522_0    conda-forge
idna                      3.1                pyhd3deb0d_0    conda-forge
imagesize                 1.3.0              pyhd8ed1ab_0    conda-forge
intel-openmp              2021.4.0          h57928b3_3556    conda-forge
ipython                   7.30.1           py39hcbf5309_0    conda-forge
jbig                      2.1               h8d14728_2003    conda-forge
jedi                      0.18.1           py39hcbf5309_0    conda-forge
jinja2                    3.0.3              pyhd8ed1ab_0    conda-forge
jpeg                      9d                   h8ffe710_0    conda-forge
kiwisolver                1.3.2            py39h2e07f2f_1    conda-forge
lcms2                     2.12                 h2a16943_0    conda-forge
lerc                      3.0                  h0e60522_0    conda-forge
libblas                   3.9.0              12_win64_mkl    conda-forge
libbrotlicommon           1.0.9                h8ffe710_6    conda-forge
libbrotlidec              1.0.9                h8ffe710_6    conda-forge
libbrotlienc              1.0.9                h8ffe710_6    conda-forge
libcblas                  3.9.0              12_win64_mkl    conda-forge
libclang                  11.1.0          default_h5c34c98_1    conda-forge
libdeflate                1.8                  h8ffe710_0    conda-forge
liblapack                 3.9.0              12_win64_mkl    conda-forge
libpng                    1.6.37               h1d00b33_2    conda-forge
libtiff                   4.3.0                hd413186_2    conda-forge
libzlib                   1.2.11            h8ffe710_1013    conda-forge
lz4-c                     1.9.3                h8ffe710_1    conda-forge
markupsafe                2.0.1            py39hb82d6ee_1    conda-forge
matplotlib                3.5.0            py39hcbf5309_0    conda-forge
matplotlib-base           3.5.0            py39h581301d_0    conda-forge
matplotlib-inline         0.1.3              pyhd8ed1ab_0    conda-forge
mkl                       2021.4.0           h0e2418a_729    conda-forge
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
numpy                     1.21.4           py39h6635163_0    conda-forge
olefile                   0.46               pyh9f0ad1d_1    conda-forge
openjpeg                  2.4.0                hb211442_1    conda-forge
openssl                   1.1.1l               h8ffe710_0    conda-forge
packaging                 21.3               pyhd8ed1ab_0    conda-forge
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    8.4.0            py39h916092e_0    conda-forge
pip                       21.3.1             pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.24             pyha770c72_0    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pygments                  2.10.0             pyhd8ed1ab_0    conda-forge
pyopenssl                 21.0.0             pyhd8ed1ab_0    conda-forge
pyparsing                 3.0.6              pyhd8ed1ab_0    conda-forge
pyqt                      5.12.3           py39hcbf5309_8    conda-forge
pyqt-impl                 5.12.3           py39h415ef7b_8    conda-forge
pyqt5-sip                 4.19.18          py39h415ef7b_8    conda-forge
pyqtchart                 5.12             py39h415ef7b_8    conda-forge
pyqtwebengine             5.12.1           py39h415ef7b_8    conda-forge
pysocks                   1.7.1            py39hcbf5309_4    conda-forge
python                    3.9.7           h7840368_3_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python_abi                3.9                      2_cp39    conda-forge
pytz                      2021.3             pyhd8ed1ab_0    conda-forge
qt                        5.12.9               h5909a2a_4    conda-forge
requests                  2.26.0             pyhd8ed1ab_1    conda-forge
setuptools                59.4.0           py39hcbf5309_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
snowballstemmer           2.2.0              pyhd8ed1ab_0    conda-forge
sphinx                    4.3.1              pyh6c4a22f_0    conda-forge
sphinxcontrib-applehelp   1.0.2                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.2                      py_0    conda-forge
sphinxcontrib-htmlhelp    2.0.0              pyhd8ed1ab_0    conda-forge
sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.3                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.5              pyhd8ed1ab_1    conda-forge
sqlite                    3.37.0               h8ffe710_0    conda-forge
tbb                       2021.4.0             h2d74725_1    conda-forge
tk                        8.6.11               h8ffe710_1    conda-forge
tornado                   6.1              py39hb82d6ee_2    conda-forge
traitlets                 5.1.1              pyhd8ed1ab_0    conda-forge
tzdata                    2021e                he74cb21_0    conda-forge
ucrt                      10.0.20348.0         h57928b3_0    conda-forge
urllib3                   1.26.7             pyhd8ed1ab_0    conda-forge
vc                        14.2                 hb210afc_5    conda-forge
vs2015_runtime            14.29.30037          h902a5da_5    conda-forge
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
wheel                     0.37.0             pyhd8ed1ab_1    conda-forge
win_inet_pton             1.1.0            py39hcbf5309_3    conda-forge
xz                        5.2.5                h62dcd97_1    conda-forge
zlib                      1.2.11            h8ffe710_1013    conda-forge
zstd                      1.5.0                h6255e5f_0    conda-forge
```

</details>

If you still can't repro from a clean env with an identical spec, that would isolate it to a platform-specific difference, or something else we're not considering.

> But I can also reproduce this on master, so it doesn't seem to be a regression from this PR.

I can't run `docrepr` successfully on `master` at all with either Sphinx 3 or Sphinx 4 due to the Sphinx issues with the extension, so I can't confirm if the problem I'm seeing is actually a regression, but if you'd rather handle this in #31 , we can go ahead and merge this one.

**Comment by martinRenou** [comment]
> Differences in our invocations python -b -X dev -c "import fabfile; fabfile.test_plot()" for me from the docrepr root dir; I get no warnings with the latest deps

It's pretty much what I do too.

> Differences in our Python environments (I'm using Python 3.9.7 from Conda-Forge)

I tried with Python 3.9.7 and it also seems to work on my side

> Differences in our dependency stacks (to confirm, I created a fresh clean environment from -c conda-forge with just docutils, jinja2, sphinx, matplotlib and ipython as specs, plus python=3.9, and the issue did not change; here's my conda list output:

I have pretty much the same conda env as you do, as much as I can see.

<details>

```
# packages in environment at /home/martin/miniconda3/envs/docrepr:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       1_gnu    conda-forge
alabaster                 0.7.12                     py_0    conda-forge
babel                     2.9.1              pyh44b312d_0    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
binutils                  2.36.1               hdd6e379_2    conda-forge
binutils_impl_linux-64    2.36.1               h193b22a_2    conda-forge
binutils_linux-64         2.36                 hf3e587d_2    conda-forge
bleach                    4.1.0              pyhd8ed1ab_0    conda-forge
brotlipy                  0.7.0           py39h3811e60_1003    conda-forge
bzip2                     1.0.8                h7f98852_4    conda-forge
c-ares                    1.18.1               h7f98852_0    conda-forge
c-compiler                1.3.0                h7f98852_0    conda-forge
ca-certificates           2021.10.8            ha878542_0    conda-forge
certifi                   2021.10.8        py39hf3d152e_1    conda-forge
cffi                      1.15.0           py39h4bc2ebd_0    conda-forge
charset-normalizer        2.0.9              pyhd8ed1ab_0    conda-forge
cmake                     3.21.3               h8897547_0    conda-forge
cmarkgfm                  0.6.0            py39h3811e60_2    conda-forge
colorama                  0.4.4              pyh9f0ad1d_0    conda-forge
compilers                 1.3.0                ha770c72_0    conda-forge
cryptography              36.0.0           py39h95dcef6_0    conda-forge
cxx-compiler              1.3.0                h4bd325d_0    conda-forge
cycler                    0.11.0             pyhd8ed1ab_0    conda-forge
dbus                      1.13.6               h48d8840_2    conda-forge
decorator                 5.1.0              pyhd8ed1ab_0    conda-forge
deprecation               2.1.0              pyh9f0ad1d_0    conda-forge
docutils                  0.17.1           py39hf3d152e_1    conda-forge
expat                     2.4.1                h9c3ff4c_0    conda-forge
flake8                    4.0.1              pyhd8ed1ab_1    conda-forge
fortran-compiler          1.3.0                h1990efc_0    conda-forge
freetype                  2.10.4               h0708190_1    conda-forge
gcc                       9.4.0                h192d537_2    conda-forge
gcc_impl_linux-64         9.4.0               h03d3576_11    conda-forge
gcc_linux-64              9.4.0                h391b98a_2    conda-forge
gettext                   0.19.8.1          h73d1719_1008    conda-forge
gfortran                  9.4.0                h2018a41_2    conda-forge
gfortran_impl_linux-64    9.4.0               h0003116_11    conda-forge
gfortran_linux-64         9.4.0                hf0ab688_2    conda-forge
glib                      2.70.2               h780b84a_0    conda-forge
glib-tools                2.70.2               h780b84a_0    conda-forge
gxx                       9.4.0                h192d537_2    conda-forge
gxx_impl_linux-64         9.4.0               h03d3576_11    conda-forge
gxx_linux-64              9.4.0                h0316aca_2    conda-forge
icu                       69.1                 h9c3ff4c_0    conda-forge
idna                      3.1                pyhd3deb0d_0    conda-forge
imagesize                 1.3.0              pyhd8ed1ab_0    conda-forge
importlib-metadata        4.2.0            py39hf3d152e_0    conda-forge
importlib_metadata        4.2.0                hd8ed1ab_0    conda-forge
ipython                   7.30.1           py39hf3d152e_0    conda-forge
jbig                      2.1               h7f98852_2003    conda-forge
jedi                      0.17.2           py39hf3d152e_2    conda-forge
jeepney                   0.7.1              pyhd8ed1ab_0    conda-forge
jinja2                    3.0.3              pyhd8ed1ab_0    conda-forge
jpeg                      9d                   h36c2ea0_0    conda-forge
jupyter-packaging         0.11.1             pyhd8ed1ab_0    conda-forge
kernel-headers_linux-64   2.6.32              he073ed8_15    conda-forge
keyring                   23.4.0           py39hf3d152e_0    conda-forge
kiwisolver                1.3.2            py39h1a9c180_1    conda-forge
krb5                      1.19.2               hcc1bbae_3    conda-forge
lcms2                     2.12                 hddcbb42_0    conda-forge
ld_impl_linux-64          2.36.1               hea4e1c9_2    conda-forge
lerc                      3.0                  h9c3ff4c_0    conda-forge
libblas                   3.9.0           12_linux64_openblas    conda-forge
libcblas                  3.9.0           12_linux64_openblas    conda-forge
libcurl                   7.80.0               h2574ce0_0    conda-forge
libdeflate                1.8                  h7f98852_0    conda-forge
libedit                   3.1.20191231         he28a2e2_2    conda-forge
libev                     4.33                 h516909a_1    conda-forge
libffi                    3.4.2                h7f98852_5    conda-forge
libgcc-devel_linux-64     9.4.0               hd854feb_11    conda-forge
libgcc-ng                 11.2.0              h1d223b6_11    conda-forge
libgfortran-ng            11.2.0              h69a702a_11    conda-forge
libgfortran5              11.2.0              h5c6108e_11    conda-forge
libglib                   2.70.2               h174f98d_0    conda-forge
libgomp                   11.2.0              h1d223b6_11    conda-forge
libiconv                  1.16                 h516909a_0    conda-forge
liblapack                 3.9.0           12_linux64_openblas    conda-forge
libnghttp2                1.43.0               h812cca2_1    conda-forge
libopenblas               0.3.18          pthreads_h8fe5266_0    conda-forge
libpng                    1.6.37               h21135ba_2    conda-forge
libsanitizer              9.4.0               h79bfe98_11    conda-forge
libssh2                   1.10.0               ha56f1ee_2    conda-forge
libstdcxx-devel_linux-64  9.4.0               hd854feb_11    conda-forge
libstdcxx-ng              11.2.0              he4da1e4_11    conda-forge
libtiff                   4.3.0                h6f004c6_2    conda-forge
libuv                     1.42.0               h7f98852_0    conda-forge
libwebp-base              1.2.1                h7f98852_0    conda-forge
libzlib                   1.2.11            h36c2ea0_1013    conda-forge
lz4-c                     1.9.3                h9c3ff4c_1    conda-forge
markupsafe                2.0.1            py39h3811e60_1    conda-forge
matplotlib                3.3.2                         0    conda-forge
matplotlib-base           3.3.2            py39h98787fa_1    conda-forge
matplotlib-inline         0.1.3              pyhd8ed1ab_0    conda-forge
mccabe                    0.6.1                      py_1    conda-forge
ncurses                   6.2                  h58526e2_4    conda-forge
nodejs                    17.1.0               h8ca31f7_2    conda-forge
numpy                     1.21.4           py39hdbf815f_0    conda-forge
olefile                   0.46               pyh9f0ad1d_1    conda-forge
openjpeg                  2.4.0                hb52868f_1    conda-forge
openssl                   1.1.1l               h7f98852_0    conda-forge
packaging                 21.3               pyhd8ed1ab_0    conda-forge
parso                     0.7.1              pyh9f0ad1d_0    conda-forge
pcre                      8.45                 h9c3ff4c_0    conda-forge
pexpect                   4.8.0              pyh9f0ad1d_2    conda-forge
pickleshare               0.7.5           py39hde42818_1002    conda-forge
pillow                    8.4.0            py39ha612740_0    conda-forge
pip                       21.3.1             pyhd8ed1ab_0    conda-forge
pkginfo                   1.8.2              pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.24             pyha770c72_0    conda-forge
ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge
pycodestyle               2.8.0              pyhd8ed1ab_0    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pyflakes                  2.4.0              pyhd8ed1ab_0    conda-forge
pygments                  2.10.0             pyhd8ed1ab_0    conda-forge
pyopenssl                 21.0.0             pyhd8ed1ab_0    conda-forge
pyparsing                 3.0.6              pyhd8ed1ab_0    conda-forge
pysocks                   1.7.1            py39hf3d152e_4    conda-forge
python                    3.9.7           hb7a2778_3_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python_abi                3.9                      2_cp39    conda-forge
pytz                      2021.3             pyhd8ed1ab_0    conda-forge
readline                  8.1                  h46c0cb4_0    conda-forge
readme_renderer           27.0               pyh9f0ad1d_0    conda-forge
requests                  2.26.0             pyhd8ed1ab_1    conda-forge
requests-toolbelt         0.9.1                      py_0    conda-forge
rfc3986                   1.5.0              pyhd8ed1ab_0    conda-forge
rhash                     1.4.1                h7f98852_0    conda-forge
secretstorage             3.3.1            py39hf3d152e_1    conda-forge
setuptools                59.4.0           py39hf3d152e_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
snowballstemmer           2.2.0              pyhd8ed1ab_0    conda-forge
sphinx                    4.3.1              pyh6c4a22f_0    conda-forge
sphinxcontrib-applehelp   1.0.2                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.2                      py_0    conda-forge
sphinxcontrib-htmlhelp    2.0.0              pyhd8ed1ab_0    conda-forge
sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.3                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.5              pyhd8ed1ab_1    conda-forge
sqlite                    3.37.0               h9cd32fc_0    conda-forge
sysroot_linux-64          2.12                he073ed8_15    conda-forge
tk                        8.6.11               h27826a3_1    conda-forge
tomlkit                   0.7.2            py39hf3d152e_0    conda-forge
tornado                   6.1              py39h3811e60_2    conda-forge
tqdm                      4.62.3             pyhd8ed1ab_0    conda-forge
traitlets                 5.1.1              pyhd8ed1ab_0    conda-forge
twine                     3.7.1              pyhd8ed1ab_0    conda-forge
tzdata                    2021e                he74cb21_0    conda-forge
urllib3                   1.26.7             pyhd8ed1ab_0    conda-forge
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
webencodings              0.5.1                      py_1    conda-forge
wheel                     0.37.0             pyhd8ed1ab_1    conda-forge
xz                        5.2.5                h516909a_1    conda-forge
yarn                      1.22.17              ha770c72_0    conda-forge
zipp                      3.6.0              pyhd8ed1ab_0    conda-forge
zlib                      1.2.11            h36c2ea0_1013    conda-forge
zstd                      1.5.0                ha95c52a_0    conda-forge
```

</details>

> Something platform specific with temp directory handling, file copying, or otherwise (since it looks like you're on *nix of some sort, while I'm on Windows)

I guess that leaves this. I am on Fedora Linux. I wonder if this issue could be fixed by https://github.com/spyder-ide/docrepr/pull/32 ?

**Comment by CAM-Gerlach** [comment]
Hmmm, thanks. Not sure, but I'll test it right now and we'll see!

I could also fire up my old Fedora VM and try to repro, if need be, though hopefully it needn't come to that...

**Comment by CAM-Gerlach** [comment]
TL;DR: No change; commented in more detail on the other PRs. Would you rather we just merge this and open a separate issue for that, or do you want to try to fix it here and/or in the other PR? Thanks!

**Comment by martinRenou** [comment]
If you don't mind, we could merge this and open a separate issue?

Then I will look into adding a CI with Github actions that runs the fabfile tests and compare the outputs with expected ones, see if we can reproduce your issue on Windows.

**Review by CAM-Gerlach — APPROVED** [review]
LGTM, thanks for your work (and patience) on this @martinRenou !
